virtio-devices/src/block.rs

```diff
@@ -754,7 +761,7 @@ impl VirtioDevice for Block {
                     })?,
                 disk_nsectors: self.disk_nsectors,
                 interrupt_cb: interrupt_cb.clone(),
-                disk_image_id: disk_image_id.clone(),
+                disk_image_id: self.serial.clone(),
```

Comment:
I'd be up for changing this member name for consistency.

Reply:
Renamed `disk_image_id` to `serial` throughout the codebase.